task2_Gal_Malka/src/Position.cpp

```diff
@@ -0,0 +1,74 @@
+#include "Position.h"
+
+Position::Position(const string& position){ 
+	row = int(position[0] - 'a');
+	col = int(position[1] - '1'); 
+}
+
+Position::Position(const int row, const int col) { 
+	this->row = row; 
```

Comment:
This is not the way to set members in C++. It should be a direct assignment without calling `this`. 
I.e - `m_row = row`